src/filters/filter_cue.cpp

```diff
@@ -351,7 +351,12 @@ namespace dmxfish::filters {
         update_last_values();
         start_time = *time;
         frame = 0;
-        cue_end_handling_real = cues.at(active_cue).end_handling;
+	if (active_cue >= cues.size()) {
+		active_cue = cues.size() - 1;
+	}
+	if(active_cue < cues.size()) {
+        	cue_end_handling_real = cues.at(active_cue).end_handling;
+	}
```

Comment:
i'm not sure if we need this this, but better its there

Reply:
As this is called rarely we could leave it in there.